BridgeApp/BridgeApp/Data Tracking/Model/SBASymptomLoggingStepObject.swift

```diff
@@ -355,3 +366,160 @@ open class SBASymptomTableItem : RSDModalStepTableItem {
         super.init(identifier: loggedResult.identifier, rowIndex: rowIndex, reuseIdentifier: reuseIdentifier)
     }
 }
+
+/// The symptom table item is tracked using the result object.
+public struct SBASymptomResult : Codable, RSDScoringResult {
+    private enum CodingKeys : String, CodingKey {
+        case identifier, text, loggedDate, timeZone, severity, duration, medicationTiming, notes
+    }
+    
+    public let type: RSDResultType = .symptom
+    
+    public let identifier: String
+    
+    /// The start date timestamp for the result.
+    public var startDate: Date = Date()
+    
+    /// The end date timestamp for the result.
+    public var endDate: Date = Date()
+    
+    /// The text shown to the user as the title.
+    public let text: String
+    
+    /// The date timestamp for when the item was logged.
+    public var loggedDate: Date?
+    
+    /// Time zone.
```

Comment:
```suggestion
    /// The time zone in effect when the item was logged.
```
(if this is indeed the intent)